src/wallet/rpcdump.cpp

```diff
@@ -651,16 +651,18 @@ UniValue z_importkey(const UniValue& params, bool fHelp)
     if (addResult == KeyNotAdded) {
         throw JSONRPCError(RPC_WALLET_ERROR, "Error adding spending key to wallet");
     }
-    
+
     // whenever a key is imported, we need to scan the whole chain
     pwalletMain->nTimeFirstKey = 1; // 0 would be considered 'no value'
-    
+
     // We want to scan for transactions and notes
     if (fRescan) {
         pwalletMain->ScanForWalletTransactions(chainActive[nRescanHeight], true);
     }
 
-    return NullUniValue;
+    auto addr = boost::apply_visitor(
+            libzcash::DefaultAddressFromSpendingKey{}, spendingkey);
+    return EncodePaymentAddress(addr);
```

Comment:
I think I'd prefer to return a JSON object that contains a `defaultAddress` field set to this (or maybe `primaryAddress` field, since Sprout is supported; alternatively we could use `address` for Sprout and `defaultAddress` for Sapling, and have a `type` field like `z_validateaddress` has). That makes it clear that this is the default address in the case of Sapling, and also means that we can extend this RPC method in future with additional information if we wanted.

Reply:
I moved the address to a `defaultAddress` field as you suggested. 

It looks like the best way to add the `type` field is to export [DescribePaymentAddressVisitor](https://github.com/zcash/zcash/blob/master/src/rpc/misc.cpp#L210), I suggest that I do that in a separate PR to limit the size of this one. What do you think?